code/game/mecha/mech_fabricator.dm

```diff
@@ -59,6 +46,29 @@
 	//building time adjustment coefficient (1 -> 0.8 -> 0.6)
 	T = -1
 	for(var/obj/item/stock_parts/manipulator/Ml in component_parts)
+		// FULPSTATION: Tiered part sets 
+		
+		part_sets = list(
+								"Cyborg","Misc"
+								)	
+		if( Ml.rating >= 2)
+			part_sets += list(
+								"Ripley","Exosuit Equipment","Exosuit Ammunition","Cyborg Upgrade Modules"

```

Comment:
I don't like how you've put "cyborg upgrade modules" in the tier 2 list (especially without mentioning that in the changelog). This is going to be really annoying for borgs who want upgrades but can't get them because Science hasn't bothered to upgrade the mech fabricators yet.

Reply:
borgs have to get upgrades from roboticists anyway. if sci hasnt upgraded the fabs the roboticists should get it to happen. tier 2 manip only costs some metal

Reply:
Yes, but you have to drag the scientists up off of their asses and make them actually upgrade robotics (if they've even printed out a BRPED or parts yet!), which will add a considerable amount of extra hassle to getting upgrades as a borg.

Also, expect roboticists to start breaking into R&D to print off BRPEDs and stock parts, then getting pissed at (and/or flashing) any scientists who try to object to their breaking and entering ("THIS IS A BASIC NECESSITY FOR MY JOB, FUCKWIT"). This is gonna be like botanists and mutagen all over again. Sure, it doesn't take much effort from a chemist to make you mutagen or from a scientist to upgrade your parts... but it's gonna take a hell of a lot of effort from you to convince them to drop what they're doing and actually give a damn about you/do a small favor that doesn't directly benefit them for you, especially if they're new and haven't ever upgraded a machine before (or are just a shitter).

Reply:
breaking into R&D? At least on box, roboticists already have access. and if its not the case they can just talk to RD or HoP to get access. i don't want to not do this PR just because you're concerned about people having to interact with each other
BRPED isnt necessary, a normal RPED can do the job. in fact, you dont need an RPED at all, you can just upgrade manually.

Reply:
"i don't want to not do this PR just because you're concerned about people having to interact with each other"
You _intentionally_ put borg upgrades in that second tier. You don't have to "not do this PR" to fix that; you can just move borg upgrades back into the first tier.

Reply:
See but theres no reason to move them from the second tier, it forces robotics to get upgrades first hand and all they cost is metal, by the time borgs are wanting upgrades, T2 is researched anyway, so there is no reason to argue this.